### ISSUE TITLE
The type element does nothing...

### ISSUE THREAD

**Issue by IngwiePhoenix** [nakupanda/bootstrap3-dialog#110]
Hey. I just copied the .js and .css from the dist/ folder and also this test code:

```javascript
var types = [BootstrapDialog.TYPE_DEFAULT,
            BootstrapDialog.TYPE_INFO,
            BootstrapDialog.TYPE_PRIMARY,
            BootstrapDialog.TYPE_SUCCESS,
            BootstrapDialog.TYPE_WARNING,
            BootstrapDialog.TYPE_DANGER];

$.each(types, function(index, type){
    BootstrapDialog.show({
        type: type,
        title: 'Message type: ' + type,
        message: 'What to do next?',
        buttons: [{
            label: 'I do nothing.'
        }]
    });
});
```

But when running that, all dialogues look the very same with zero difference... What am I doing wrong? Thus, can I apply an extra class to push the dialogue more towards the center of the page instead of the very top?

**Comment by cmwelsh** [comment]
Your example code worked for me using latest Bootstrap CSS and default theme CSS. The background color of the title changed to match each type.

**Comment by IngwiePhoenix** [comment]
I figured out what was the case, and i just had to go all „duh“. When downloading, I used `curl -O $url` on the links that you get when you right-click the „Raw“ button in a source file view. Turns out that it contained a redirect - and since I did not pass in -L, I got the HTML output instead, which broke the CSS. The JS I downloaded after I had accidentially clicked the Raw link - so only the CSS got a bit disturbed. It now works and I am working on implementing stuff like prompt and alike for my project. :)

Very great plugin!
